torch/_higher_order_ops/effects.py

```diff
@@ -22,6 +22,10 @@ class _EffectType(Enum):
 }
 
 
+def register_side_effect_op(op: torch._ops.OpOverload) -> None:
```

Comment:
haha i also added this [here](https://github.com/pytorch/pytorch/pull/122348/files#diff-f5c5ba29e62f89dc3d91301b1c5af37e0bc43b90fee45458dfddac0d217103c2R25). I guess we can see who lands first and go with whichever :>

Reply:
Sure!